components/project-card/meta/get-involved.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class GetInvolved extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  handleGetInvolvedLinkClick() {
+    this.props.sendGaEvent(`Get involved`, `Clicked`, `beacon`);
+  }
+
+  renderGetInvolvedText() {
+    let props = this.props;
+    let getInvolvedText = props.getInvolved ? props.getInvolved : null;
+    let getInvolvedLink = props.getInvolvedUrl ? ( <a href={props.getInvolvedUrl} target="_blank" onClick={() => this.handleGetInvolvedLinkClick()}>Get Involved</a>) : null;
+
+    if (!getInvolvedText && !getInvolvedLink) return null;
+
+    return <p>{getInvolvedText} {getInvolvedLink}</p>;
+  }
+
+  renderHelpLabels() {
+    if (!this.props.helpTypes) return null;
+
+    return this.props.helpTypes.map(helpType => {
+      // TODO:FIXME: links to `/help/${encodeURIComponent(helpType)}`
```

Comment:
is there an issue for this already? If so, instead of simply noting it needs to be done, point to that issue. If there isn't, create an issue and then update the comment =)